plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQuerySplitManager.java

```diff
@@ -117,6 +149,15 @@ public ConnectorSplitSource getSplits(
         return new FixedSplitSource(splits);
     }
 
+    private TableId createDestinationTable(TableId remoteTableId)
```

Comment:
`create` looks a little misleading because it doesn't create tables actually. How about `createDestinationTable` or something? 

nit: This method can be static.